### DISCUSSION TITLE
Entering my DB Confi - Gets stuck in loop

### DISCUSSION THREAD

**Discussion by lighthousewebc** [akeeba/panopticon#354] [Q&A (Community Support)]
So I have made up a new cpanel account for this and made up a new Database name and User name
- database type - MySQLi
- Database server host name - localhost
- User Name - Made one up
- Password - Enter it in
- Database Name - Made one up and linked the USER TO DB
- Database table name prefix - left as defualt

Then I click on NEXT and it just goes in a loop and goes back to the Welcome to Panopticon 
 - LET GO button 
 - ALL SYSTEMS GO
 - Your server meets all requirements. You may continue installing Panopticon.
 - NEXT
 - Goes back to Data Config page

I use your Akeeba backup all the time and I know when something is messed up in the Name or password you give that ERROR msg.. 

Thanks looking forward to try this out

Akeeba Panopticon 1.0.5

**Comment by nikosdion** [comment]
Try using PDO MySQLm and double check all the database information you are using.

**Comment by lighthousewebc** [comment]
Ok I did this

1. Delete ALL DB and USERS
2. Remade DB and User
3. Did what you told me it worked
4. clicked the next button 
5. then I got to the place to enter in my name and email address
6. Then it said it was done 
7. Now its back in the LOOP again with the Database Configuration page
/index.php?view=setup&task=database&layout=database
8. I see in the DB it made up some of the Tables
- but i noticed that it DIDN"T save my user that I created Table is blank

9, So I was doing my install in Chrome
Then I tired FF and still won't get me pass the enter in Name and email and username - then gets in the loop

I'm using cPanel
MySQL Version 10.6.16-MariaDB
php 8.2

**Comment by lighthousewebc** [comment]
I'm keep trying to get access so I did this for a test

1. DELETE EVERYTHING
2. Reinsatlled the zip file
3. make the htaccess.txt and renamed it to .htaccess
4. make up a new DB and User and password and used the PDO
5. Then go to the Enter in NAME Page and password
6. Put my name in 
7. Put my own user name in
8. Password
9. TIme zone change to Winnipeg
10. STUCK IN LOOP
11. Then I tried the this
12. Name - But my name in 
13. User - Left as ADMIN
14. passsord - put in something
15. UTC - left default
16. Now I get to the /index.php?view=setup&task=cron
17. Now I get error msg
`Benchmark failed
There was an error which prevented the benchmark from completing.

	Akeeba Panopticon

		{"login.url":"https:\/\/akeeba.WEBSITE.com\/index.php?view=login&lang="}
		
					svg.panopticonLogoColour {height: 6em;margin-bottom: 1em;}
	
	    path.main{fill:black} path.w{fill:#73888c}}
        
            Please log in        
    
            Username
   
            Password
       
            Login        

                Login Language            
            
	🇳🇱 Nederlands – Nederland
	🇬🇧 English – United Kingdom
	🇬🇷 Ελληνικά – Ελλάδα
	🇩🇪 Deutsch - Deutschland
	🇮🇹 Italiano – Italia
	🇫🇷 Français – Françe
	🇭🇷 Hrvatski
`

**Comment by lighthousewebc** [comment]
So now I try to go and log back in 
1. It brings up the main login page
2. So I enter in admin 
3. and my password
4. I will not allow me in
5. THought I entered in the password wrong so I go to the DB like I do if I forget my MASTER ADMIn for Joomla 
 and try to update the password the same way and it will not allow me in 

If you want access to the cPanel account let me know and you can see for your Testing what is going on for a LIVE account

**Comment by nikosdion** [comment]
I have not been able to reproduce your issue. As long as you enter the correct database information, Panopticon installs and can log in after installation just fine.

I don't understand what "stuck in a loop" is supposed to express in your description, but I don't think it means what you think it means. If something is stuck in a loop, it repeats the same executable code, consuming all resources. There is no indication you have something like this.

The best guess I can make is that you are installing in a subdirectory or subdomain of an existing site, with the Panopticon root folder under another site's root folder. This would cause all sorts of chaos due to .htaccess shenanigans.

Since this is not something anyone else has had in these past two weeks, or anything I can reproduce, I don't know how else to help you. Sorry.
